NastassiaBon/hometask-4/my_hash/my_hash_RSpec.rb

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative 'my_hash'
+require 'rspec'
+require 'pry'
+
+RSpec.describe MyHash do
+  subject(:my_hash) { described_class.new }
```

Comment:
it's a bad idea to have one global variable for all tests, better to use dedicated for each one